addons/website/views/website_visitor_views.xml

```diff
@@ -202,6 +202,7 @@
                     </div>
                     <group id="general_info">
                         <group string="Visitor Informations">
+                            <field name="active" invisible="0"/>
```

Comment:
Why invisible="0", this is the default value right ?

Reply:
@tivisse , It was mistake. Done the Changes. thanks